llvm/include/llvm/CodeGen/TargetInstrInfo.h

```diff
@@ -766,6 +766,22 @@ class TargetInstrInfo : public MCInstrInfo {
     createTripCountGreaterCondition(int TC, MachineBasicBlock &MBB,
                                     SmallVectorImpl<MachineOperand> &Cond) = 0;
 
+    /// Create a condition to determine if the remaining trip count for a phase
+    /// is greater than TC. Some instructions such as comparisons may be
+    /// inserted at the bottom of MBB. The all instructions expanded for the
```

Comment:
"The all instructions" -> "All instructions"

Reply:
Fixed